data/sql/updates/pending_db_world/rev_1679899428135820100.sql

```diff
@@ -0,0 +1,45 @@
+-- EffectMiscValueB_1 64 does not make the adds show the title "'s minion"
+UPDATE `spell_dbc` SET `Effect_1` = 28, `EffectMiscValueB_1` = 67 WHERE `ID` IN (36042, 36043, 36044, 36045, 36046, 36047, 36048, 36049, 36050);
```

Comment:
It seems like TC has all those spells with MiscValue set to 64, though

![image](https://user-images.githubusercontent.com/100873791/228970662-ea4a826b-6894-440f-ac6b-ad2bec7a672e.png)

Reply:
Yeah but 64 is doesn't classify them as a "minion", look at source.

Reply:
uhg, using 67 does not allow more than 1 summon (since they're counted as a pet)